app/lib/actions/katello/agent_action.rb

```diff
@@ -0,0 +1,106 @@
+module Actions
+  module Katello
+    class AgentAction < Actions::EntryAction
+      include Dynflow::Action::Timeouts
+      include Helpers::Presenter
+
+      def self.agent_message
+        fail NotImplementedError
+      end
+
+      def agent_action_type
+        nil
```

Comment:
so nil agent action type is ok?

Reply:
Yeah, see how it's used in DispatchHistoryPresenter